runbooks/how-to-resolve-snyk-report-findings.md

```diff
@@ -0,0 +1,16 @@
+Follow this runbook for each SNYK finding that appears in the
+[SNYK dashboard](https://app.snyk.io/org/bluebutton-fd-oeda) that does not have an associated SNYK bot PR automatically
+created in github.
```

Comment:
"..., for those findings that _do_ have a SNYK bot PR, follow the ____ runbook."